.eslintrc.js

```diff
@@ -1,19 +1,22 @@
 module.exports = {
   root: true,
   parser: '@typescript-eslint/parser',
-  plugins: [
-    '@typescript-eslint',
-    'jest'
-  ],
+  plugins: ['@typescript-eslint', 'jest'],
   rules: {
-    "@typescript-eslint/no-explicit-any": "off"
+    '@typescript-eslint/member-delimiter-style': [
+      'warn',
+      {
+        multiline: { delimiter: 'none' },
```

Comment:
What does this do?

Reply:
It's to remove the need for delimiters in interfaces and types definitions, causing the change you commented on below.
I had to make this change in the ESLint config because I have Prettier configured to remove trailing semi-columns.